### ISSUE TITLE
crash on testing suite when not run from bin directory

### ISSUE THREAD

**Issue by Ulrond** [rdkcentral/ut-core#31]
looks like assets folder issues

```bash
./bin/ut-test -b
```

gives

```bash
2024-05-03-10:36:52, ERROR , src/ut_kvp.c,   180 : No Data File open
2024-05-03-10:36:52, LOG   , src/cunit/cunit_lgpl/ut_basic.c,   284 : Suite ut-kvp - assert testing yaml , Test kvp assert uint64 had failures:
2024-05-03-10:36:52, LOG   , src/cunit/cunit_lgpl/ut_basic.c,   291 :     1. tests/src/ut_test_kvp_assert.c:48  - CU_ASSERT_EQUAL(ut_kvp_getUInt64Field( ut_kvp_assert_getInstance(), "decodeTest/checkUint64IsDeadBeefDec" ),checkField)
024-05-03-10:36:52, LOG   , src/cunit/cunit_lgpl/ut_basic.c,   300 :      Test Complete : 'kvp assert uint64'
2024-05-03-10:36:52, LOG   , src/cunit/cunit_lgpl/ut_basic.c,   252 :      Running Test : 'kvp assert string'
2024-05-03-10:36:52, ERROR , src/ut_kvp.c,   359 : No Data File openSegmentation fault (core dumped)
```

**Comment by kanjoe24** [comment]
also crashes on make run

`make -C tests/ run`

**Comment by Ulrond** [comment]
Fixed in release Milestone 2.1.3 - #36
